### ISSUE TITLE
Error running script

### ISSUE THREAD

**Issue by ilanbiala** [toleda/audio_CloverALC#60]
```
$ ./audio_cloverALC-110_v1.0p10.command
./audio_cloverALC-110_v1.0p10.command: line 1: a#!/bin/sh: No such file or directory

Agreement
The audio_cloverALC-110 script is for personal use only. Do not distribute
the patch, any or all of the files or the resulting patched AppleHDA.kext
for any reason without permission. The audio_cloverALC-110 script is
provided as is and without any kind of warranty.

File: audio_cloverALC-110.command_v1.0p10
Password:
EFI partition not mounted
Crucial/EFI not found
EFI/CLOVER folder not available to install audio
No system files were changed
To save a Copy of this Terminal session: Terminal/Shell/Export Text As ...

ilan at IBIALA in ~/Downloads
$ ./audio_cloverALC-110_v1.0p10.command
./audio_cloverALC-110_v1.0p10.command: line 1: a#!/bin/sh: No such file or directory

Agreement
The audio_cloverALC-110 script is for personal use only. Do not distribute
the patch, any or all of the files or the resulting patched AppleHDA.kext
for any reason without permission. The audio_cloverALC-110 script is
provided as is and without any kind of warranty.

File: audio_cloverALC-110.command_v1.0p10
EFI partition is mounted
Boot/Arguments = kext-dev-mode=1 found
Confirm Realtek ALC892 (y/n): y
Clover Audio ID Injection (y/n): y
Use Audio ID: 1 (y/n): y

Download ALC892 files ...
Confirm Realtek ALC892 test (y/n): y
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     9  100     9    0     0     36      0 --:--:-- --:--:-- --:--:--    36
[/tmp/ALC892.zip]
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of /tmp/ALC892.zip or
        /tmp/ALC892.zip.zip, and cannot find /tmp/ALC892.zip.ZIP, period.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  2707  100  2707    0     0  12286      0 --:--:-- --:--:-- --:--:-- 12304
Edit config.plist/Devices/Audio/Inject/1
Edit config.plist/SystemParameters/InjectKexts/YES
Download kext patches
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1404  100  1404    0     0   6457      0 --:--:-- --:--:-- --:--:--  6440
Install /Volumes/EFI/EFI/CLOVER/kexts/10.10/realtekALC.kext
Install System/Library/Extensions/AppleHDA.kext/ALC892 zml files
install: /tmp/892/Platforms.xml.zlib: No such file or directory
install: /tmp/892/layout1.xml.zlib: No such file or directory
install: /tmp/892/layout2.xml.zlib: No such file or directory
install: /tmp/892/layout3.xml.zlib: No such file or directory
Error: Installation failure
rm: /System/Library/Extensions/AppleHDA.kext/Contents/Resources/*zml.zlib: No such file or directory
Original S/L/E/AppleHDA.kext restored
To save a Copy of this Terminal session: Terminal/Shell/Export Text As ...
```

A file isn't found and it causes the audio script to not install.

**Comment by toleda** [comment]
Apologies, fixed in v1.0p20

**Comment by AlbertEmil** [comment]
@toleda: If so, shouldn't this issue be closed?
